tests/zfs-tests/tests/functional/rsend/rsend_024_pos.ksh

```diff
@@ -49,8 +49,8 @@ streamfs=$POOL/stream
 
 log_onexit resume_cleanup $sendfs $streamfs
 
-test_fs_setup $sendfs $recvfs
-log_must zfs unmount $sendfs
+test_fs_setup $sendfs $recvfs $streamfs
+log_must zfs unmount -f $sendfs
```

Comment:
This is from e3bdcb8ad88e7a90726193c9afe96a713729c928, not aee1dd4d983c64db3c3155290d48f05243e85709: maybe we should also add that commit to the stack?

Reply:
Sharp eye. I'll remove that.